diagnostic.md

```diff
@@ -35,5 +46,7 @@ Record your responses inside the fenced code blocks below each question.
     pass it data?
 
     ```html
-    <!-- your response here -->
+    {{#each model.my-phone as |phone|}}
+      {{my-contact/my-phone phone=phone}}
+    {{/each}}
```

Comment:
you will access the object property via `contact.phones` instead of `model.my-phone`, and you don't need `my-contact/`.  So like this:
```
{{#each contact.phones as |phone|}}
{{my-phone phone=phone}}
{{/each}}
```